tools/poctools/makefile

```diff
@@ -0,0 +1,22 @@
+PYTHON=.venv/bin/python
+PIP=.venv/bin/pip
+TWINE=.venv/bin/twine
+
+.venv:
+	python -m venv .venv
+	${PIP} install -r requirements.txt
+
+/dist/: build
+/build/: build
+
+build: .venv
+	${PYTHON} setup.py sdist bdist_wheel
+
+publish: /dist/ /build/
+	${TWINE} upload dist/*
```

Comment:
Trengs det ikke noe autentisering av noe slag?

Reply:
Når du kjører denne kommandoen så blir du bedt om brukernavn og passord. Jeg tenkte å utsette det å sette opp secrets frem til jeg deployer en ny versjon. Ser ikke for meg at det kommer til å skje med det første

Reply:
@yngvark tror du det er innafor, eller skulle jeg ha gått opp den veien nå

Reply:
Neida det funker fint.